.clang-format

```diff
@@ -0,0 +1,40 @@
+BasedOnStyle: WebKit
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: Consecutive
+AlignConsecutiveDeclarations: Consecutive

```

Comment:
I suggest setting `AlignConsecutiveDeclarations: false` it removes the big empty whitespaces (stack struct as an example.) when the type declaration is too long for the other consecutive assigments.